#a point on the line
float64 pointX
float64 pointY
float64 pointZ

#the direction of the line as vector
float64 directionX
float64 directionY
float64 directionZ

#the number of points matched in the point cloud
int32 pointCount

#the extracted point cloud of the matched line
sensor_msgs/PointCloud2 cloud
